assets/blocks/reader-registration/edit.js

```diff
@@ -11,14 +11,31 @@ import { intersection } from 'lodash';
 import apiFetch from '@wordpress/api-fetch';
 import { __ } from '@wordpress/i18n';
 import { useState, useEffect } from '@wordpress/element';
-import { Spinner, Notice, TextControl, ToggleControl, PanelBody } from '@wordpress/components';
-import { RichText, useBlockProps, InspectorControls } from '@wordpress/block-editor';
+import {
+	useBlockProps,
+	RichText,
+	InspectorControls,
+	useInnerBlocksProps,
+	InnerBlocks,
+} from '@wordpress/block-editor';
+import {
+	Spinner,
+	Notice,
+	TextControl,
+	ToggleControl,
+	PanelBody,
+	Button,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
 
+const variantOptions = [
```

Comment:
NIT: `variant` does not feel like a proper name for this. How about `blockState`?

Reply:
Fixed in 2ae772000da5724b4a4589e2ee1f8132e7be2d2b